driver-core/src/main/com/mongodb/operation/AsyncChangeStreamBatchCursor.java

```diff
@@ -40,7 +40,7 @@
     AsyncChangeStreamBatchCursor(final ChangeStreamOperation<T> changeStreamOperation,
                                  final AsyncBatchCursor<RawBsonDocument> wrapped,
                                  final AsyncReadBinding binding) {
-        this.changeStreamOperation = changeStreamOperation;
+        this.changeStreamOperation = changeStreamOperation.startAtOperationTime(binding.getSessionContext().getOperationTime());
```

Comment:
This shouldn't replace a startAtOperationTime supplied by the application, or be set if the application supplied a resumeToken.

Reply:
Done